pkg/registry/servicecatalog/broker/strategy_test.go

```diff
@@ -61,3 +94,57 @@ func TestBroker(t *testing.T) {
 		t.Fatalf("Fresh broker should have empty status")
 	}
 }
+
+func TestValidateUpdateStatusPrepareForUpdate(t *testing.T) {
+	cases := []struct {
+		name                string
+		old                 *sc.Broker
+		newer               *sc.Broker
+		shouldChecksum      bool
+		checksumShouldBeSet bool
+	}{
+		{
+			name:                "not ready -> not ready",
```

Comment:
I can go do this later and write down the wrong answer, but if @jpeeler is writing the test now, and understands what he wrote, I'd rather he did it.